exporters/trace/jaeger/jaeger_test.go

```diff
@@ -319,19 +319,23 @@ func TestExporter_ExportSpan(t *testing.T) {
 
 	assert.NoError(t, err)
 
-	tp := sdktrace.NewTracerProvider(
-		sdktrace.WithSampler(sdktrace.AlwaysSample()),
-		sdktrace.WithSyncer(exp),
-	)
+	opts := append(exp.o.TracerProviderOptions, sdktrace.WithSyncer(exp))
+	tp := sdktrace.NewTracerProvider(opts...)
```

Comment:
This can also be accomplished by using `NewExportPipeline` above instead of `NewRawExporter`.  Since this test isn't explicitly for `NewRawExporter` and as some people use tests as a form of documentation of idiom, it might be better to use that function to create `tp` directly.

Reply:
It seems that if I use `NewExportPipeline`, I would have to setup the environment variables just like what `TestNewExporterPipelineWithOptions` does. I wonder if I can leave the `TestExporter_ExportSpan` as it is and make the tag assertion in `TestNewExporterPipelineWithOptions` ?

Reply:
Sure, that works too.  It looks like that test is already checking `GetServiceName()` so you'd just need to add some tags to check as well.

Reply:
I just added the tag assertions, could you plz take a look?

Reply:
LGTM 🚀